infoschema/test/clustertablestest/tables_test.go

```diff
@@ -1811,3 +1811,16 @@ func TestAddFieldsForBinding(t *testing.T) {
 	require.Equal(t, rows[0][7], "use_index(@`sel_1` `test`.`t` ), ignore_index(`t` `a`)")
 	require.Equal(t, rows[0][8], "select * from `t` where `a` = ?")
 }
+
+func TestCheckConstraints(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("SET GLOBAL tidb_enable_check_constraint = ON")
+	tk.MustExec("CREATE TABLE test.t1 (id INT PRIMARY KEY, CHECK (id<10))")
+	rows := tk.MustQuery("SELECT * FROM information_schema.CHECK_CONSTRAINTS").Rows()
```

Comment:
Could you please add a case about what's the `SELECT * FROM information_schema.CHECK_CONSTRAINTS ` result after dropping a constraint

Reply:
Done